### ISSUE TITLE
Large shapefile fail to import

### ISSUE THREAD

**Issue by Kjellis85** [domlysz/BlenderGIS#459]
# **Blender and OS versions**

Blender 2.90.1, Windows 10 Enterprise 20H2

# **Describe the bug**

Shapefileimport fails to import, creates logfile

# **How to Reproduce**

Import a large contour shapefile (2gb)

# **Error message**

Blender 2.90.1, Commit date: 2020-09-23 06:43, Hash 3e85bb34d0d7
bpy.ops.object.delete(use_global=False, confirm=False)  # Operator
Deleted 1 object(s)  # Info
bpy.ops.object.delete(use_global=False, confirm=False)  # Operator
Deleted 1 object(s)  # Info
bpy.ops.object.delete(use_global=False, confirm=False)  # Operator
Deleted 1 object(s)  # Info
bpy.context.space_data.system_bookmarks_active = 1  # Property
bpy.ops.object.origin_set(type='ORIGIN_GEOMETRY', center='MEDIAN')  # Operator

backtrace
Exception Record:

ExceptionCode         : EXCEPTION_ACCESS_VIOLATION
Exception Address     : 0x00007FF6ACECEE28
Exception Module      : blender.exe
Exception Flags       : 0x00000000
Exception Parameters  : 0x2
	Parameters[0] : 0x0000000000000001
	Parameters[1] : 0x0000000000000410

**Comment by domlysz** [comment]
Yes this function is not memory friendly, all the dataset is loadeded into the RAM; Blender itself is not designed to handle very large geodataset. You should make concessions here.
